### PR TITLE
Feature/make api

### PR DESCRIPTION
**Pull request by Matt-Roden**
# Description drawing

- Added an apiCalls file
- Successfully made a fetch and set the data into state in App

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactor feature (non-breaking change that modifies existing work)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How Has This Been Tested?

- [x] local host 3000
- [x] console.log()
- [x] dev tools
- [ ] cypress testing

# Checklist:

- [x] My code follows the style guidelines of this project/linted
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,26 +1,31 @@
...
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+class App extends Component {
```
Comment (hayleyw7) [inline src/App.js:6]: It works!

File: src/apiCalls.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+export const getPokedexData = () => {
```
Comment (hayleyw7) [inline src/apiCalls.js:1]: I'm glad we got this started straight in the apiCalls.

File: src/App.js [hunk 3]
```diff
@@ -1,26 +1,31 @@
...
+        pokeDex: data.results
+      })
+    })
+  }
+
+  render() {
+    return(
+      <div>Hello World</div>
+    )
+  }
```
Comment (joanafbrito) [inline src/App.js:28]: This was a really good start!

File: src/apiCalls.js [hunk 4]
```diff
@@ -0,0 +1,4 @@
...
+  return fetch('https://pokeapi.co/api/v2/pokemon?limit=151')
+    .then(res => res.ok ? res.json() : this.displayErrorInfo(res))
+}
```
Comment (joanafbrito) [inline src/apiCalls.js:4]: Love this code!

**Review by hayleyw7 — APPROVED** [review]
Looks just as we left it: working and great progress!

**Review by joanafbrito — APPROVED** [review]
Great job putting all this together

**Pull request merged**